proofOfConcept/ColorPickerAlpha/ColorPickerAlpha/MainWindow.xaml

```diff
@@ -26,36 +28,36 @@
 
         </Grid.RowDefinitions>
 
-        <Button Grid.Row = "0" Grid.Column = "0"  Click="Button_Click" Opacity="1" VerticalAlignment="Center" HorizontalAlignment="Center" Height="40" Width="46"  Margin="10,4,10,5" >
+        <Button Grid.Row = "0" Grid.Column = "0" Opacity="1" VerticalAlignment="Center" HorizontalAlignment="Center" Height="40" Width="46"  Margin="10,4,10,5" >
             <Button.Background>
                 <ImageBrush ImageSource="Eyedropper-icon.png" Stretch="Uniform"/>
             </Button.Background>
         </Button>
 
-        <Rectangle Grid.Row = "0" Grid.Column = "1" Width="40" Height="40" Fill="White" Stroke="Black" />
-        
+        <Rectangle x:Name="Color_Box" Grid.Row = "0" Grid.Column = "1" Width="40" Height="40" Fill="White" Stroke="Black"/>
```

Comment:
Consider using system colours {ThemeResource Something} to avoid problems with light/dark mode and more importantly accessibility like high contrast, etc...